Trustroots/trustroots
# Revert #1056 - Remove endpoint for runtime extraction of missing translations

#### Proposed Changes

Revert all changes introduced in #1056, except extracted translations and some i18n config. Why? Because when #1291 gets merged, we don't need this anymore.

#### Testing Instructions

* `git switch revert-1056-remove-i18next-backend`
* see that the app still works

Reverts #1056 
Follow up of #1291
